components/licensor/ee/pkg/licensor/licensor.go

```diff
@@ -162,6 +163,24 @@ func NewEvaluator(key []byte, domain string) (res *Evaluator) {
 	}
 }
 
+func matchesDomain(pattern, domain string) bool {
+	if pattern == "" {
+		return true
+	}
```

Comment:
Does this check imply that a license without domain is valid on any domain? 

We should not allow licenses without domain, because if such licenses exists we have no control over who's using them.

Reply:
Yes, but this is not new behaviour! The "default" license works just in that way.